src/cameras/repository/query.js

```diff
@@ -0,0 +1,15 @@
+import { gql } from '@apollo/client'
+
+export const GET_ALL_CAMERAS = gql`
```

Comment:
Since this query is going to be filtered, the `GET_ALL_CAMERAS` name is a lie. Let's drop the `ALL` part.

Reply:
done